Assets/_Game/Scripts/Fungus/ItemGiveFromDB.cs

```diff
@@ -0,0 +1,31 @@
+using UnityEngine;
+using Fungus;
+
+[CommandInfo("DLS",
+             "Item Give From DB",
+             "Gives an Item from the DB to another Inventory")]
+[AddComponentMenu("")]
+public class ItemGiveFromDB : Command
+{
+    [SerializeField] protected ItemDatabaseObject databaseSource;
+    [SerializeField] protected InventoryObject inventoryTarget;
+    [SerializeField] protected string itemName;
+    [SerializeField] protected int itemAmount;
+    public override void OnEnter()
+    {
+        if(databaseSource != null && inventoryTarget != null)
+        {
+            var item = databaseSource.GetItem[itemName];

```

Comment:
You know my stance on var

Reply:
The never ending stance between explicit and implicit conventions I get your stance but still var LOL